x/interchainstaking/keeper/ibc_packet_handlers.go

```diff
@@ -362,12 +362,25 @@ func (k *Keeper) HandleMsgTransfer(ctx sdk.Context, msg sdk.Msg) error {
 		return errors.New("unexpected recipient")
 	}
 
-	return k.HandleDistributeFeesFromModuleAccount(ctx)
-}
+	zone := k.GetZoneForWithdrawalAccount(ctx, sMsg.Sender)
+
+	receivedCoin := sMsg.Token
+	if zone != nil && receivedCoin.Denom != zone.BaseDenom {
+		feeAmount := sdk.NewDecFromInt(receivedCoin.Amount).Mul(k.GetCommissionRate(ctx)).TruncateInt()
+		rewardCoin := receivedCoin.SubAmount(feeAmount)
+		zoneAddress, err := addressutils.AccAddressFromBech32(zone.WithdrawalAddress.Address, "")
+		if err != nil {
+			return err
+		}
+		k.Logger(ctx).Info("distributing collected rewards to users", "amount", rewardCoin)
+		err = k.BankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, zoneAddress, sdk.NewCoins(rewardCoin)) // Fee collector name needs to be passed in to keeper constructor.
```

Comment:
What does this comment mean?

Reply:
No idea. I think it was pre-existing. Will remove.